pm_muairss_write/get_out_folder.py

```diff
@@ -0,0 +1,27 @@
+import os
+import sys
+
+import yaml
+
+
+def filefinder(root, substr, exception='', exceptfile=''):
+    '''finds files with names containing substr
+        within folder root and its subfolders'''
+    filelst = []
+    for i in os.walk(root):
+        for j in i[2]:
+            if (substr in j and (exceptfile == '' or exceptfile not in j) and
+                    (exception == '' or exception not in i[0])):
+                filelst.append(i[0]+os.sep+j)
+    return filelst
+
+
+try:
+    with open('params.yaml') as f:
+        data = yaml.safe_load(f)
+except FileNotFoundError:
+    sys.stdout.write('default')
```

Comment:
1. Why the choice of `sys.stdout.write` over `print` here? I'm unfamiliar with the former - does it just avoid a trailing `\n` or are there other reasons to use it?
2. If `out_folder` is not provided, `pm-muairss -t w` [defaults to](https://github.com/muon-spectroscopy-computational-project/pymuon-suite/blob/master/pymuonsuite/schemas.py#L173) the folder name `muon-airss-out`, so this line should reflect that. Currently it causes an error as no folder `default` exists to put into the zip.
3. We should add a test that includes a parameter file that does not include `out_folder` to catch this problem in future.